logstash-core-event/spec/logstash/timestamp_spec.rb

```diff
@@ -106,4 +107,50 @@
       expect(subject.to_f).to eq(now.to_f)
     end
   end
+
+  context "at" do
```

Comment:
can we add potentially 3 more tests?

1. arg==nil
2. arg < 0
3. arg != numeric type